linchpin/cli/__init__.py

```diff
@@ -432,7 +455,8 @@ def lp_destroy(self, targets=(), run_id=None, tx_id=None):
                                     tx_id=tx_id)
 
 
-    def _execute_action(self, action, targets=(), run_id=None, tx_id=None):
+    def _execute_action(self, action, targets=(), run_id=None, tx_id=None,
+                        old_schema=False):
```

Comment:
What's the purpose of adding `old_schema` here? The util.dataparser does the conversion from the old schema to the new without indicating anything here. I wonder if maybe having the validator run *after* the conversion might be better?

Reply:
The old_schema flag needs to make its way down to the _validate_topology method.  If a document cannot be validated with either schema, this flag determines under which schema error will be reported